frontend/src/app/shared/models/match.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Roster } from "./roster.model";
+
+export class Match {
+  public id: number;
+  public teamA: Roster;
+  public teamB: Roster;
```

Comment:
Do we want teamA and teamB associated with Roster or LeagueMember? Makes sense atm because we don't have a model for LeagueMember, but something to consider.